platforms/hyperledger-indy/charts/indy-node/templates/configmap.yaml

```diff
@@ -10,4 +10,18 @@ metadata:
     app.kubernetes.io/instance: {{ .Release.Name }}
 data:
   indy_config.py: |  
-  {{ $.Values.configmap.indyConfig  | nindent 6 }}
\ No newline at end of file
+  {{ $.Values.configmap.indyConfig  | nindent 6 }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ $.Values.node.name }}-domain-transactions-genesis
+  namespace: {{ $.Values.metadata.namespace }}
+  labels:
+    app.kubernetes.io/name: {{ $.Values.node.name }}-domain-transactions-genesis
```

Comment:
all nodes within one org use the same config map

Reply:
using organization name now